google/cloud/aiplatform/models.py

```diff
@@ -215,10 +228,534 @@ def upload(
+            resource_name=endpoint_name,
+            resource_noun="endpoints",
+            project=self.project,
+            location=self.location,
+        )
+        endpoint = self.api_client.get_endpoint(name=endpoint_name)
+
+        return endpoint
+
+    @classmethod
+    def create(
+        cls,
+        display_name: str,
+        description: Optional[str] = None,
+        labels: Optional[Dict] = None,
+        metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> "Endpoint":
+        """Creates a new endpoint.
+
+        Args:
+            display_name (str):
+                Required. The user-defined name of the Endpoint.
+                The name can be up to 128 characters long and can be consist
+                of any UTF-8 characters.
+            description (str):
+                Optional. The description of the Endpoint.
+            labels (Dict):
+                Optional. The labels with user-defined metadata to
+                organize your Endpoints.
+                Label keys and values can be no longer than 64
+                characters (Unicode codepoints), can only
+                contain lowercase letters, numeric characters,
+                underscores and dashes. International characters
+                are allowed.
+                See https://goo.gl/xmQnxf for more information
+                and examples of labels.
+            metadata (Sequence[Tuple[str, str]]):
+                Optional. Strings which should be sent along with the request as
+                metadata.
+            project (str):
+                Optional. Project to retrieve endpoint from. If not set, project
+                set in aiplatform.init will be used.
+            location (str):
+                Optional. Location to retrieve endpoint from. If not set, location
+                set in aiplatform.init will be used.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials to use to upload this model. Overrides
+                credentials set in aiplatform.init.
+        Returns:
+            endpoint (Endpoint):
+                Instantiated representation of the endpoint resource.
+        """
+
+        api_client = cls._instantiate_client(location=location, credentials=credentials)
+
+        create_endpoint_operation = cls._create(
+            api_client=api_client,
+            parent=initializer.global_config.common_location_path(
+                project=project, location=location
+            ),
+            display_name=display_name,
+            description=description,
+            labels=labels,
+            metadata=metadata,
+        )
+
+        endpoint = cls(project=project, location=location, credentials=credentials)
+
+        lro.LRO.update_resource(
```

Comment:
To me it seems like there is bug here because this is not a staticmethod:
https://github.com/googleapis/python-aiplatform/blob/dev/google/cloud/aiplatform/lro.py#L59

I think `LRO.update_resource` needs to be changed to take self and not an operation future.

And then this should be `create_endpoint_operation.update_resource` without operation future passed in.